lib.rs

```diff
@@ -1504,6 +1504,24 @@ pub struct IntoIter<A: Array> {
     end: usize,
 }
 
+/*impl<A: Array + Clone> Clone for IntoIter<A>
```

Comment:
You can remove this. Since we already have a specialization for `SmallVec::from`, we won't need a separate specialized impl here.

Reply:
I'm just having a doubt about this: if I just keep the below impl, won't `IntoIter<A>::clone()` only work where `A::Item: Clone`? I don't know how Rust plans to implement specialization, but won't the impl which limits `A::Item` to types which implement `Clone` then prevent `Copy` types to call the less-expensive `From` impl?

Reply:
Or do you mean we just provide one impl and modify it when const generics land to work well with all possible `From` implementations?

Reply:
Right.